astropy/cosmology/flrw.py

```diff
@@ -298,29 +271,19 @@ def Ok0(self):
         at z=0"""
         return self._Ok0
 
-    @property
-    def Tcmb0(self):
-        """ Temperature of the CMB as `~astropy.units.Quantity` at z=0"""
-        return self._Tcmb0
-
     @property
     def Tnu0(self):
         """ Temperature of the neutrino background as `~astropy.units.Quantity` at z=0"""
         return self._Tnu0
 
-    @property
-    def Neff(self):
-        """ Number of effective neutrino species"""
-        return self._Neff
-
     @property
     def has_massive_nu(self):
         """ Does this cosmology have at least one massive neutrino species?"""
         if self._Tnu0.value == 0:
             return False
         return self._massivenu
 
-    @property
+    @m_nu.getter
```

Comment:
I think it should be relatively straightfoward to be able to do `@Parameter` here. Would that be nicer?

Reply:
It definitely would be, but I like defining all the Parameters in one place. Easier to chunk mentally.

Reply:
No reason not to define it as part of that list, I think. But anyway, a detail!